requirements/requirements.txt

```diff
@@ -1,7 +1,7 @@
 colorama
 distrax @ git+https://github.com/google-deepmind/distrax  # distrax release doesn't support jax > 0.4.13
```

Comment:
I'm not sure, but there hasn't been a Distrax release since November 2023. So I think we should just keep things as is for now and investigate later.

Reply:
Ah this is a relic. I commented on their repo around that time and they released the fix, it used to just be on their github

Reply:
Okay so then I am changing it to just distrax?